src/controllers/midi/midicontroller.cpp

```diff
@@ -537,3 +585,36 @@ void MidiController::processInputMapping(const MidiInputMapping& mapping,
     qCWarning(m_logBase) << "MidiController: No script function specified for"
                          << MidiUtils::formatSysexMessage(getName(), data, timestamp);
 }
+
+const QString sErrorMessage(
+        "Invalid timer callback provided to midi.makeInputHandler. "
+        "Please pass a function and make sure that your code contains no syntax errors.");
```

Comment:
these belong in an anonymous namespace (our pattern to achieve internal linkage) at the top of the file. Also prefix them `k` for constant instead of `s` for static and wrap the string in a QStringLiteral.
```suggestion
const QString kMakeInputHandlerError = QStringLiteral(
        "Invalid timer callback provided to midi.makeInputHandler. "
        "Please pass a function and make sure that your code contains no syntax errors.");
```